_ml-commons-plugin/gpu-acceleration.md

```diff
@@ -0,0 +1,303 @@
+---
+layout: default
+title: GPU acceleration
+parent: Model-serving framework
+nav_order: 150
+---
+
+
+# GPU acceleration 
+
+GPU acceleration is an experimental feature. For updates on the progress of GPU acceleration, or if you want to leave feedback that could help improve the feature, join the discussion in the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+When running a natural language processing (NLP) model in your OpenSearch cluster with a machine learning (ML) node, you can achieve better performance on the ML node using graphics processing unit (GPU) acceleration. GPUs can work in tandem with the CPU of your cluster to speed up the model upload and training. 
+
+## Supported GPUs
+
+Currently, ML nodes support any GPU instances that use CUDA 11.6 or greater, including the following:
+
+- [NVIDIA G-series instances](https://aws.amazon.com/nvidia/)
+- [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/)
+
+If you need GPU power, you can provision GPU instances through [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/). For more information on how to provision a GPU instance, see [Recommended GPU Instances](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).
+
+## Supported images
+
+You can use GPU acceleration with both [Docker images](https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/supported-tags.md) with CUDA 11.6 and [Amazon Machine Images (AMIs)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
+
+## PyTorch
+
+GPU-accelerated ML nodes require [PyTorch](https://pytorch.org/docs/stable/index.html) 1.12.1 or greater to work with ML models.
+
+## Setting up a GPU-accelerated ML node
+
+Depending on the GPU, you can provision a GPU-accelerated ML node manually or by using automated initialization scripts. 
+
+### Mounting an nvidia-uvm device
```

Comment:
This script is from https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#device-node-verification, should we use the same title "Device node verification"?

Reply:
Sorry, just realize this part is at the same level with `### Preparing AWS Inferentia`, then I think we should change to `### Preparing NVIDIA GPU ML node`